server/views/certificate/data-vis.jade

```diff
@@ -1,5 +1,5 @@
 meta(name='viewport', content='width=device-width, initial-scale=1')
-link(rel='stylesheet', href='/bower_components/bootstrap/dist/css/bootstrap.min.css')
+link(rel='stylesheet', href='http://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css')
```

Comment:
Gtk. What do you think about configuring webpack to serve css files from `node_modules`, rather than use cdn.